invenio_indexer/api.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         record_cls=None,
         record_dumper=None,
         publish_kwargs=None,
+        bulk_chunk_size=500,
```

Comment:
```suggestion
        bulk_index_max_items=10_000,
```

- nit: renamed, to possibly avoid confusion with anything passed to the ES/OS bulk index helper.
- minor: shouldn't this by default be something relatively high? Otherwise, since [500 is also the default chunk size](https://github.com/elastic/elasticsearch-py/blob/3f1d5498eddb5b0aec1cce28a5a96177202bb673/elasticsearch/helpers/actions.py#L361) for the bulk index, the task would just make one request/batch and exit immediately. With 10k it'll be 20 requests.

Reply:
All solved, I understood from our chat that you were aiming at having 1 task per bulk batch, that's why the name was chosen on purpose to make it as ES.
It makes a lot of sense to me to put in much higher :)